File: src/main/resources/frontend/rfgcp/src/app/@core/model/detalhe-conta.ts
export class DetalheConta {
  nome: string;
  email: string;
  cashPoint: string;
  ultimoLogon: Date;
  ultimoLogoff: Date;
  statusPersonagem: boolean;
  dataFinalPremium: Date;
  premium: boolean;

  constructor(nome?: string, email?: string, cashPoint?: string, ultimoLogon?: Date,
              ultimoLogoff?: Date, statusPersonagem?: boolean, dataFinalPremium?: Date,
              premium?: boolean) {
    this.nome = nome;
    this.email = email;
    this.cashPoint = cashPoint;
    this.ultimoLogon = ultimoLogon;
    this.ultimoLogoff = ultimoLogoff;
    this.statusPersonagem = statusPersonagem;
    this.dataFinalPremium = dataFinalPremium;
    this.premium = premium;
  }
}

File: src/main/java/br/com/rfreforged/ReforgedGCP/model/usuario/DetalheConta.java
package br.com.rfreforged.ReforgedGCP.model.usuario;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

@AllArgsConstructor
@NoArgsConstructor
@Data
public class DetalheConta {
    private String nome;
    private String email;
    private double cashPoint;
    private Date ultimoLogon;
    private Date ultimoLogoff;
    private boolean statusPersonagem;
    private Date dataFinalPremium;
    private boolean premium;

    @Override
    public String toString() {
        return "DetalheConta{" +
                "nome='" + nome + '\'' +
                ", email='" + email + '\'' +
                ", cashPoint=" + cashPoint +
                ", ultimoLogon=" + ultimoLogon +
                ", ultimoLogoff=" + ultimoLogoff +
                ", statusPersonagem=" + statusPersonagem +
                ", dataFinalPremium=" + dataFinalPremium +
                ", premium=" + premium +
                '}';
    }
}
